pkg/kubelet/cm/node_container_manager_linux.go

```diff
@@ -157,6 +158,10 @@ func getCgroupConfig(rl v1.ResourceList) *ResourceConfig {
 		val := MilliCPUToShares(q.MilliValue())
 		rc.CpuShares = &val
 	}
+	if q, exists := rl[pidlimit.PIDs]; exists {
+		val := q.Value()
+		rc.PodPidsLimit = &val
```

Comment:
can we rename `ResourceConfig.PodPidsLimit` to `ResourceConfig.PidsLimit` since it isn't pod-specific anymore?

Reply:
OK.
I'm currently working on an e2e test for this.

Reply:
Just pushed a node e2e for it.  Not entirely sure that there's anything worth unit testing; all of the interesting stuff is in the cgroup manipulation.